apps/web-giddh/src/app/contact/account-statement/account-statement.component.html

```diff
@@ -0,0 +1,444 @@
+                                <th mat-header-cell *matHeaderCellDef class="bg-black text-white font-16 fw-900">
+                                    {{ localeData?.details }}
+                                </th>
+                                <td
+                                    mat-cell
+                                    *matCellDef="let element; let i = index"
+                                    [ngClass]="{ 'bg-light-grey': i % 2 }"
+                                    class="details-column word-break"
+                                >
+                                    {{ element?.voucherNumber ?? "-" }}
+                                </td>
+                            </ng-container>
+                            <ng-container matColumnDef="Amount">
+                                <th
+                                    mat-header-cell
+                                    *matHeaderCellDef
+                                    class="bg-black text-white sort-header-right text-right font-16 fw-900"
+                                >
+                                    {{ commonLocaleData?.app_amount }}
+                                </th>
+                                <td
+                                    mat-cell
+                                    *matCellDef="let element; let i = index"
+                                    [ngClass]="{ 'bg-light-grey': i % 2 }"
+                                    class="sort-header-right text-right"
+                                >
+                                    <span
+                                        >{{
+                                            element?.voucherAmount.type === "DEBIT"
+                                                ? element?.voucherAmount?.amount
+                                                : "-"
+                                        }}
+                                    </span>
+                                </td>
+                            </ng-container>
+                            <ng-container matColumnDef="Payments">
+                                <th
+                                    mat-header-cell
+                                    *matHeaderCellDef
+                                    class="bg-black text-white sort-header-right text-right font-16 fw-900"
+                                >
+                                    {{ commonLocaleData?.app_payment }}
+                                </th>
+                                <td
+                                    mat-cell
+                                    *matCellDef="let element; let i = index"
+                                    [ngClass]="{ 'bg-light-grey': i % 2 }"
+                                    class="sort-header-right text-right"
+                                >
+                                    {{ element?.voucherAmount.type != "DEBIT" ? element?.voucherAmount?.amount : "-" }}
```

Comment:
```suggestion
                                    {{ element?.voucherAmount.type !== "DEBIT" ? element?.voucherAmount?.amount : "-" }}
```

Use === use same code everywhere as pr giddh format